core/pta/veraison_attestation/veraison_attestation.c

```diff
@@ -0,0 +1,158 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/*
+ * Copyright (C) 2024, Institute of Information Security (IISEC)
+ */
+
+#include <base64.h>
+#include <kernel/pseudo_ta.h>
+#include <mempool.h>
+#include <pta_veraison_attestation.h>
+#include <stdlib.h>
+#include <string.h>
+
+#include "cbor.h"
+#include "hash.h"
+
+#define PTA_NAME "veraison_attestation.pta"
+
+#define MAX_KEY_SIZE 4096
+#define MAX_NONCE_SIZE 64
+#define TEE_SHA256_HASH_SIZE 32
+
+#define EAT_PROFILE "http://arm.com/psa/2.0.0"
+#define CLIENT_ID 1
+#define LIFECYCLE 12288
+#define MEASURMENT_TYPE "PRoT"
+#define SIGNER_ID_LEN 32
+#define INSTANCE_ID_LEN 33
+
+/* clang-format off */
+#define SIGNER_ID (                                        \
+	0xac, 0xbb, 0x11, 0xc7, 0xe4, 0xda, 0x21, 0x72,    \
+	0x05, 0x52, 0x3c, 0xe4, 0xce, 0x1a, 0x24, 0x5a,    \
+	0xe1, 0xa2, 0x39, 0xae, 0x3c, 0x6b, 0xfd, 0x9e,    \
+	0x78, 0x71, 0xf7, 0xe5, 0xd8, 0xba, 0xe8, 0x6b     \
+)
+#define INSTANCE_ID (                                      \
+	0x01, 0xce, 0xeb, 0xae, 0x7b, 0x89, 0x27, 0xa3,    \
+	0x22, 0x7e, 0x53, 0x03, 0xcf, 0x5e, 0x0f, 0x1f,    \
+	0x7b, 0x34, 0xbb, 0x54, 0x2a, 0xd7, 0x25, 0x0a,    \
+	0xc0, 0x3f, 0xbc, 0xde, 0x36, 0xec, 0x2f, 0x15,    \
+	0x08                                               \
+)
```

Comment:
Can't use parentheses here, see 'CI / make (multi-platform)' error report . Prefer use braces here and remove them from lines 60/61.

Reply:
OK, thank you.

Reply:
I fixed it, was the build successful? I have confirmed that it builds in my local environment.